docs/pyproject.toml

```diff
@@ -9,8 +9,8 @@ package-mode = false
 [tool.poetry.dependencies]
 python = "^3.12"
 sphinx = "^7.2.6"
-jax = "^0.4.25"
-jaxlib = "^0.4.25"
+jax = "0.4.29"
+jaxlib = "^0.4.29"
```

Comment:
Numpyro 0.15.1 is released and makes this unnecessary https://github.com/pyro-ppl/numpyro/releases/tag/0.15.1